group1/comm/templates/comm/index.html

```diff
@@ -93,7 +93,7 @@ <h4 class="sidebar_description">Quick links</h4>
             <button id="upload file" class="btn btn-default" type="button" data-toggle="tooltip" title="Send files!" onclick="filechoose()">
               <span class="glyphicon glyphicon-folder-open" aria-hidden="true"></span>
             </button>
-            <button id="sendcleartext" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick="display();">
+            <button id="sendcleartext" class="btn btn-default" type="button" data-toggle="tooltip" title="Start Video Share!" onclick = "startVideoChat()">
```

Comment:
To be consistent with the rest of the html in this file, can you remove the spaces surrounding "=" for the onclick assignment?